hasura/metadata/databases/default/tables/public_users.yaml

```diff
@@ -188,6 +188,22 @@ event_triggers:
       paused: true
       schedule: 0 0 * * *
       timeout: 60
+  - name: insertOrgMember
+    definition:
+      enable_manual: false
+      insert:
+        columns: '*'
+      update:
```

Comment:
if we undelete a circle user, (UDPATE) then we want to re-add them to the org. so this is correct.